src/modules/reading-room/components/KeywordsFilterForm/KeywordsFilterForm.tsx

```diff
@@ -102,6 +97,10 @@ const KeywordsFilterForm: FC<KeywordsFilterFormProps> = ({ children, className }
 
 	const onKeyDown = (e: KeyboardEvent<HTMLInputElement>) => onKey(e, [...keysEnter], saveInput);
 
+	const components = {
+		...TAGS_INPUT_COMPONENTS,
+		DropdownIndicator: () => <div className="u-pr-8" />,
+	};
```

Comment:
Is er een reden dat deze in het component moet?
`DropdownIndicator` gaat bij elke re-render veranderen in memory

Reply:
Ik kreeg een error dat een component gebruikt werd voor die gedefinieerd was.
Maar ik heb het nog eens geprobeerd er terug buiten te zetten en het issue doet zich niet meer door. Ik pas het aan.